v22.1/cost-based-optimizer.md

```diff
@@ -158,6 +158,22 @@ We strongly recommend not setting this value higher than 8 to avoid performance
 
 For more information about the difficulty of selecting an optimal join ordering, see our blog post [An Introduction to Join Ordering](https://www.cockroachlabs.com/blog/join-ordering-pt1/).
 
+### Reduce planning time for plans with many joins
+
+The cost-based optimizer explores multiple join orderings to find the lowest-cost plan. If there are many joins or join subtrees in the query, this can increase the number of execution plans the optimizer explores, and therefore the exploration and planning time. If the plan phase of a query takes a long time (on the order of multiple seconds or minutes) to plan, or the query plan involves many joins, consider the following alternatives to reduce the planning time:
+
+- To limit the size of the subtree that can be reordered, set the `reorder_joins_limit` session variable to a lower value, for example:
+
+    ~~~ sql
+    SET reorder_joins_limit = 2;
+    ~~~
+
+    If the ordering is acceptable, for the shortest planning time, you can set `reorder_joins_limit` to `0`.
```

Comment:
nit: "If the **join** ordering **inherent in the query** is acceptable ..."

nit: If we want to be extra clear we can add a sentence like "This disables exploration of join orderings entirely."